algorithms/tree/unique_BT.py

```diff
@@ -0,0 +1,57 @@
+"""
+Given the Number of Nodes of Binary Tree.
+Return the Number of Unique Binary Tree.
+
+Algo:
+1. Find the Factorial of n(i.e Number of Nodes).
+2. Find the Catalan Number of n.
+More About Catalan Number: https://en.wikipedia.org/wiki/Catalan_number#First_proof
+3. Multiply Both of Them to calculate the Number of Unique Number of Binary Tree.
+"""
+def factorial(n):
+    '''
+    Function to find the Factorial.
+    :param n: Number of Nodes.
+    :return: Factorial of Number.
+    '''
+    fact = 1
+    for i in range(1,n+1):
+        fact *= i
+    return fact
+def binomial_coefficient(n, k):
```

Comment:
nit: i believe `factorial` `catalan_number` and `binomial_coefficient` belong to `maths` package. 
do you think it makes sense to put these in `maths` and import them in here?

Reply:
We have Only Factorial in Built Function in Math Module.
We Don't Have Catalan Number and Binomial Function in Module.
https://bugs.python.org/issue35431 (ISSUE FOR BINOMIAL FUNCTION IN Python Bug Tracker)

Reply:
sorry, I meant the math module in this package. `algorithms.maths`. I was suggesting just copy and paste the content in the new file in algorithms.maths.